tasks/service-runner/app/workers/pull-worker.ts

```diff
@@ -25,6 +28,23 @@ const LOCKFILE_DIR = '/tmp';
 // retry twice for tests and 1200 (2 minutes) for real
 const maxPrimeRetries = process.env.NODE_ENV === 'test' ? 2 : 1_200;
 
+// Exponential back-off retry delay between failed requests
+axiosRetry(axios, { 
+  // delay in milliseconds = (2^retryNumber) * 100
+  retryDelay: (retryNumber) => exponentialDelay(retryNumber + exponentialDelayOffest),
+  retryCondition: 
+    (error) => {
+      if (isNetworkOrIdempotentRequestError(error) || error.code === 'ECONNABORTED') {
+        logger.error('Axios retry condition has been met.',
```

Comment:
Do we have a 'warn' log level? Might be better than 'error' here.

Reply:
Sure, I'll change that.